core/input/input.cpp

```diff
@@ -34,6 +34,7 @@
 #include "core/input/default_controller_mappings.h"
 #include "core/input/input_map.h"
 #include "core/os/os.h"
+#include "servers/display_server.h"
```

Comment:
This was discussed in a PR-review meeting and Input should not depend on display_server.
An alternative approach would be to notify Input from the main loop about application focus changes.